### PR TITLE
Edited GeneratePathExamples.cs

### PR DESCRIPTION
**Pull request by naveen965**

### CODE REVIEW

File: Assets/Scripts/SphereCollid.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    /*private void OnTriggerEnter(Collider other)
+    {
+        TileManager tileManager = FindObjectOfType<TileManager>();
+
+        if (other.name == "3DBall")
+        {
+            Debug.Log("Colleced!");
+        }
+    }*/
+
```
Comment (senaveen5) [inline Assets/Scripts/SphereCollid.cs:28]:
```suggestion

Comment (senaveen5) [reply Assets/Scripts/SphereCollid.cs:28]: Remove this code lines

Comment (naveen965) [reply Assets/Scripts/SphereCollid.cs:28]: Ok sure

**Review by senaveen5 — COMMENTED** [review]
Fine

**Review by senaveen5 — APPROVED** [review]
Good

**Pull request merged**